### DISCUSSION TITLE
Re-triggering the bodies to map announcement

### DISCUSSION THREAD

**Discussion by JamesRabe** [EDCD/EDDI#2499] [EDDI VoiceAttack Q&A]
How do get EDDI & VA to re-announce the bodies worth mapping in a system?  If I want to map them later, I must make a note, as it won't remind me again.  (Make a note or DSS a body.  Then it will re-trigger the 'other things here worth mapping'.)

**Comment by Tkael** [comment]
I believe it should re-trigger if you do another honk in that system. ;-)

**Reply by JamesRabe** [reply]
Re-honking isn't triggering it.
If I scan a planet, then that will trigger it.
Is there a change I can make to System Scan Complete module to trigger it on another FSS scan?

**Reply by Tkael** [reply]
Re-honking should re-trigger it, provided:
1) You are using the default script,
2) Either 
  a) your ship is using the "Exploration" role
  b) your ship is using the "Multipurpose" role and you are in a system with no factions,
3) All bodies in the system have already been scanned, and
4) There are bodies in the system which meet the requirements of the `Bodies to map` script

![image](https://user-images.githubusercontent.com/4382539/213610476-f26c3f76-1cd9-47d4-bb7f-4570985eefb1.png)

**Reply by JamesRabe** [reply]
1) True
2) Don't think a cutter is flagged as exploration.  System has factions.
3) FSS is 100%
4) Yes there are, as demonstrated by DSS a low value moon, etc. and triggering the bodies to map module.

Looks like I may need to modify my discovery scan module to flag all my ships, not just exploration or multipurpose (in a system with no factions).  How do you recommend I best do that?  Don't want to take a sledgehammer to it when a simple change would be more elegant.

Hm, elegant might be to check if I have a DSS equipped?

REM'ing out:
    {if ship.Role.edname = "Exploration" || 
        (ship.Role.edname = "Multipurpose" && len(reportSystem.factions) = 0):
would be more heavy-handed I would think, but might be the easiest solution.

Or maybe
FROML (ship.Role.edname = "Multipurpose" && len(reportSystem.factions) =0):
TO:       (ship.Role.edname = "Multipurpose" && len(reportSystem.factions) > -1):

**Reply by Tkael** [reply]
You assign ship roles from the shipyard tab.

**Reply by JamesRabe** [reply]
Thanks!  That is a simple solution, but it did take me while to figure out you didn't mean shipyard in game.
